.github/workflows/frontend_charm_integration_tests.yml

```diff
@@ -1,45 +1,34 @@
-name: Run charm tests
+name: Run charm integration tests
 on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
perhaps we don't need this as it triggers whenever someone reviews the PR

Reply:
Turned out I was missing an `if` condition as per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-when-a-pull-request-is-approved (and as implemented in the metabox tests from github.com/canonical/checkbox where I copied this from)